components/BasicExampleView/ProjectCodes.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useCallback, useMemo } from 'react';
+import { ProjectFile, ProjectFolder, ProjectCode } from '../BasicExampleProjects';
+import { CodeBlock, Icon } from '@/components';
+import classNames from 'classnames';
+
+function SubProjectComponent({
+  component,
+  onClickFile,
+}: {
+  component: Array<ProjectFile | ProjectFolder> | undefined;
+  onClickFile: (projectFile: ProjectFile) => void;
+}) {
+  if (!component) {
+    return null;
+  }
+  return (
+    <ul className="folder_sub_list" style={{ display: 'block' }}>
+      {component.map((child) => {
+        return (
+          <li className={classNames('folder_item', { is_active: child.isFile && child.isOpen })} key={child.name}>
+            <button
+              type="button"
+              className="btn_folder"
+              onClick={() => {
+                if (child.isFile) {
+                  onClickFile(child);
+                }
+              }}
+            >
+              {child.isFile ? <Icon type="file" /> : <Icon type="folder" />}
+              <span className="name">{child.name}</span>
+              {/* TODO: nested folder does not work */}
+              {/* {!child.isFile && child.children && <SubProjectComponent component={child.children} />} */}
```

Comment:
Issue 1. (CSS) Nested project folder does not indent properly

![Screen Shot 2022-12-16 at 10 37 01 AM](https://user-images.githubusercontent.com/57612141/208001994-d0d558dd-9d7a-4e2f-b476-454ae7e6d61c.png)

In the sidebar, you can see that the project codes can be examined. However, the style for nested folders seems to not work well. There seems to be an update in yorkie-ui.